src/components/BackButton/BackButton.tsx

```diff
@@ -0,0 +1,26 @@
+import { StyledBackButton } from './StyledBackButton';
+import { ButtonProps } from '../Button/Button';
+import { useNavigate } from 'react-router';
+
+export interface BackButtonProps extends ButtonProps {
+  text?: string;
+  path?: string;
+}
+export const BackButton = ({
+  text = 'Back To Home',
+  path = '/',
+  ...restprops
+}: BackButtonProps) => {
+  const navigate = useNavigate();
+  return (
+    <StyledBackButton
+      {...restprops}
+      onClick={() => {
+        navigate(path);
+      }}
+      variant="dark"
```

Comment:
See previous comment
```suggestion
```

Reply:
I'm gonna keep the variant because BackButton uses Button, so it needs to define the variant. In `StyledBackButton`, you'll see `styled(Button)`